orchagent/notifications.cpp

```diff
@@ -24,5 +26,5 @@ void on_switch_shutdown_request()
     /* TODO: Later a better restart story will be told here */
     SWSS_LOG_ERROR("Syncd stopped");
```

Comment:
Should we remove this TODO and dummy log?

Reply:
TODO: Hmmm!! Not sure. May be. Let me drop.

Reply:
I will keep the log, which is at the point of receiving shutdown. The new one added, appears at the end of the loop.

Reply:
TODO dropped.